src/Iris/Tests/Tactics.lean

```diff
@@ -115,6 +115,43 @@ theorem multiple_patterns [BI PROP] (Q : PROP) : ⊢ □ (P1 ∧ P2) -∗ Q ∨
 
 end intro
 
+-- revert
+namespace revert
+
+theorem spatial [BI PROP] (P : PROP) (H : ⊢ P -∗ P) : P ⊢ P := by
+  iintro HP
+  irevert HP
+  exact H
+
+theorem intuitionistic [BI PROP] (P : PROP) (H : ⊢ □ P -∗ P) : □ P ⊢ P := by
+  iintro □HP
+  irevert HP
+  exact H
+
+theorem pure [BI PROP] (P : PROP) (Hφ : φ) : ⊢ (⌜φ⌝ -∗ P) -∗ P := by
+  iintro H
+  irevert Hφ
+  iexact H
+
+theorem «forall» [BI PROP] (Φ : α → PROP) : ⊢ (∀ x, Φ x) → Φ x := by
```

Comment:
I was quite confused by this test since I did not see where `x` comes from. I think it would be good to add it as an explicit parameter to the theorem and not have it implicitly quantified.